src/middlewared/middlewared/etc_files/systemd.py

```diff
@@ -10,7 +10,9 @@
 async def render(service, middleware):
     services = []
     services_enabled = {}
-    for service in await middleware.call("datastore.query", "services.services", [], {"prefix": "srv_"}):
+    for service in await middleware.call(
+        "datastore.query", "services.services", [["service", "nin", ["kubernetes"]]], {"prefix": "srv_"}
```

Comment:
We we are treating kubernetes as a special service, why does it have to be in services table?

Reply:
We could either show that as Applications in the services table or let the user know it's k8s, so if you think we should maybe rename this to Applications, we can do that. 
Why we need to have it in services table is that not every user will want it and for ones who do, can start / stop / enable it then as desired.

Reply:
Well, dont think it should be a configurable service? Like right now we have Plugins and we cant just turn off plugins. That would be odd. Either you install them or you dont